payment_lyra/helpers/constants.py

```diff
@@ -7,108 +7,205 @@
 # Copyright: Copyright © Lyra Network
 # License:   http://www.gnu.org/licenses/agpl.html GNU Affero General Public License (AGPL v3)
 
-from odoo import _
-
-# WARN: Do not modify code format here. This is managed by build files.
-LYRA_PLUGIN_FEATURES = {
-    'multi': True,
-    'restrictmulti': False,
-    'qualif': False,
-    'shatwo': True,
-}
-
 LYRA_PARAMS = {
-    'GATEWAY_CODE': 'Lyra',
-    'GATEWAY_NAME': 'Lyra Collect',
-    'BACKOFFICE_NAME': 'Lyra Expert',
-    'SUPPORT_EMAIL': 'support-ecommerce@lyra-collect.com',
-    'GATEWAY_URL': 'https://secure.lyra.com/vads-payment/',
-    'SITE_ID': '12345678',
-    'KEY_TEST': '1111111111111111',
-    'KEY_PROD': '2222222222222222',
-    'CTX_MODE': 'TEST',
-    'SIGN_ALGO': 'SHA-256',
-    'LANGUAGE': 'en',
-
-    'GATEWAY_VERSION': 'V2',
-    'PLUGIN_VERSION': '1.2.1',
-    'CMS_IDENTIFIER': 'Odoo_10-14',
+    "GATEWAY_CODE": "Lyra",
+    "GATEWAY_NAME": "Lyra Collect",
+    "BACKOFFICE_NAME": "Lyra Expert",
+    "SUPPORT_EMAIL": "support-ecommerce@lyra-collect.com",
+    "GATEWAY_URL": "https://secure.lyra.com/vads-payment/",
+    "SITE_ID": "12345678",
+    "KEY_TEST": "1111111111111111",
+    "KEY_PROD": "2222222222222222",
+    "CTX_MODE": "TEST",
+    "SIGN_ALGO": "SHA-256",
+    "LANGUAGE": "en",
+    "GATEWAY_VERSION": "V2",
+    "PLUGIN_VERSION": "2.0.0",
+    "CMS_IDENTIFIER": "Odoo_15",
 }
 
+# https://payzen.io/en-EN/form-payment/reference/vads-language.html
```

Comment:
this comments: https://github.com/lyra/plugin-odoo/issues/15#issuecomment-1012255533 make me realized that it should use those data https://docs.lyra.com/fr/collect/form-payment/standard-payment/vads-language.html  instead